### PR TITLE
[JENKINS-29922] marking a meta step as such

### PR DESCRIPTION
**Pull request by kohsuke**
let's hold the merge until all the changes for JENKINS-29922 is ready

### CODE REVIEW

**Comment by kohsuke** [comment]
This PR and other [JENKINS-29922](https://issues.jenkins-ci.org/browse/JENKINS-29922) PRs are ready to merge: @reviewbybees

**Comment by abayer** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -121,7 +121,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>junit</artifactId>
-            <version>1.9</version>
+            <version>1.15-SNAPSHOT</version>
```
Comment (jglick) [inline pom.xml]: :bug: use 1.15

**Comment by kohsuke** [comment]
Redeployed every jar in related plugins.

**Comment by svanoort** [comment]
🐝

File: src/test/java/org/jenkinsci/plugins/workflow/steps/CoreStepTest.java [hunk 2]
```diff
@@ -45,9 +48,11 @@
 
     @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
     @Rule public JenkinsRule r = new JenkinsRule();
+    private SnippetizerTester st = new SnippetizerTester(r);
 
     @Test public void artifactArchiver() throws Exception {
         WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "p");
+        // TODO 2.7.x+ use core symbols
         p.setDefinition(new CpsFlowDefinition("node {writeFile text: '', file: 'x.txt'; step([$class: 'ArtifactArchiver', artifacts: 'x.txt', fingerprint: true])}", true));
```
Comment (abayer) [inline src/test/java/org/jenkinsci/plugins/workflow/steps/CoreStepTest.java]: Ah, that answers my question from https://github.com/jenkinsci/workflow-cps-plugin/pull/28. =)

**Comment by abayer** [comment]
:bee:

**Comment by kohsuke** [comment]
:bee:

**Pull request merged**